### PR TITLE
[WebProfilerBundle] do not install FrameworkBundle `6.4.0-BETA1`

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Comment by derrabus** [comment]
Shall we have FrameworkBundle 6.4 conflict with WebProfilerBundle 5.4 instead? Otherwise, Composer might decide to pin WebProfilerBundle 5.4 to an old patch release.

**Comment by xabbuh** [comment]
> Shall we have FrameworkBundle 6.4 conflict with WebProfilerBundle 5.4 instead? Otherwise, Composer might decide to pin WebProfilerBundle 5.4 to an old patch release.

We have in BETA2, but we didn't have one in BETA1. Maybe we should simply conflict with BETA1?

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

File: src/Symfony/Bundle/WebProfilerBundle/composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/config": "^4.4|^5.0|^6.0",
-        "symfony/framework-bundle": "^5.3|^6.0",
+        "symfony/framework-bundle": "^5.3|^6.0,<6.4",
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/WebProfilerBundle/composer.json:21]: I updated the way the constraint is expressed, but this means the same

**Pull request merged**